src/ui/components/pages/Assets.tsx

```diff
@@ -3,72 +3,99 @@ import cn from 'classnames';
 import * as React from 'react';
 import { useState } from 'react';
 import { ActiveAccountCard } from '../accounts/activeAccountCard';
-import { Trans, useTranslation } from 'react-i18next';
+import { Trans } from 'react-i18next';
 import {
-  getAliases,
   getAsset,
   getBalances,
-  getNfts,
-  getTxHistory,
   setActiveAccount,
   setUiState,
 } from '../../actions';
 import { PAGES } from '../../pageConfig';
 import { Asset, Money } from '@waves/data-entities';
-import { Modal, Tab, TabList, TabPanel, TabPanels, Tabs } from '../ui';
+import {
+  Button,
+  BUTTON_TYPE,
+  Input,
+  Modal,
+  Tab,
+  TabList,
+  TabPanels,
+  Tabs,
+} from '../ui';
 import { Intro } from './Intro';
 import { FeatureUpdateInfo } from './FeatureUpdateInfo';
 import { useAppDispatch, useAppSelector } from 'ui/store';
-import { BigNumber } from '@waves/bignumber';
-import { AssetItem } from './assets/assetItem';
-import { NftItem } from './assets/nftItem';
 import { AssetInfo } from './assets/assetInfo';
-import { HistoryItem } from './assets/historyItem';
-import {
-  ITransaction,
-  WithId,
-} from '@waves/waves-transactions/dist/transactions';
+import { TabAssets } from './assets/tabs/tabAssets';
+import { TabNfts } from './assets/tabs/tabNfts';
+import { TabTxHistory } from './assets/tabs/tabTxHistory';
 
-const MONTH = [
-  'Jan',
-  'Feb',
-  'Mar',
-  'Apr',
-  'May',
-  'Jun',
-  'Jul',
-  'Aug',
-  'Sep',
-  'Oct',
-  'Nov',
-  'Dec',
-];
+export function SearchInput({ value, onInput, onClear }) {
+  const input = React.createRef<Input>();
+
+  return (
+    <div className={cn('flex grow', styles.searchInputWrapper)}>
+      <Input
+        ref={input}
+        className={cn(styles.searchInput, 'font300')}
+        onInput={onInput}
+        value={value}
+        spellCheck={false}
+      />
+      {value && (
+        <Button
+          className={styles.searchClose}
+          type={BUTTON_TYPE.CUSTOM}
+          onClick={() => {
+            input.current.focus();
+            onClear();
+          }}
+        >
+          <svg
+            width="14"
+            height="14"
+            viewBox="0 0 18 18"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <rect
```

Comment:
It's an empty rect used to size icons, which can be removed. And by the way, `xmlns` is not needed on `svg` when it's not a separate svg file.